umh-core/examples/example-config-redpanda.yaml

```
@@ -0,0 +1,28 @@
+
+redpanda:
+  name: redpanda
+  desiredState: active
+  redpandaServiceConfig:
+    # Note: The below settings only apply to newly created topics
+
+    # this corresponds to log_retention_ms (See also: https://docs.redpanda.com/current/manage/cluster-maintenance/disk-utilization/)
+    defaultTopicRetentionMs: 604800000
+
+    # this corresponds to retention_bytes (See also: https://docs.redpanda.com/current/manage/cluster-maintenance/disk-utilization/)
+    defaultTopicRetentionBytes: 0
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)